bbc/simorgh
# Call mPulse bundle using UserContext

Resolves #1630

**Overall change:** Call mPulse bundle using UserContext

Behaviour <a href="https://media.giphy.com/media/dTuk8wcL9XryNrKtjZ/source.mp4" target="_blank"> 
https://media.giphy.com/media/dTuk8wcL9XryNrKtjZ/source.mp4
</a>

**Code changes:**

- Removes mPulse snippet from head
- Calls it if/when user has accepted personalisation cookies
- Creates UserContext which holds info about bbc cookie

How to test: 
Enable mpulse here https://github.com/bbc/simorgh/blob/mPulseCookieLock-Context/src/app/lib/config/toggles/index.js#L10

Use a real api key here https://github.com/bbc/simorgh/blob/mPulseCookieLock-Context/envConfig/local.env#L6

Mpulse should only fire when 
`the toggle is enabled`
`the api key is set to any value`
`on client`
`user has accepted personalisation cookie`

The beacons shouldnt fire untill the user has accepted the cookie banner. If they have and they refresh, they should fire instantly. It should fire a beacon for every page view and in-simorgh navigation (article to article)



---
